docs/supervisor.md

```
@@ -0,0 +1,40 @@
+
+```bash
+curl -H "Authorization: Bearer $HACK_GATEWAY_TOKEN" \
+  -H "Content-Type: application/json" \
+  -X POST \
+  http://127.0.0.1:7788/control-plane/projects/$PROJECT_ID/jobs \
+  -d '{"runner":"generic","command":["bash","-lc","echo hello"]}'
+```
+
+Then connect to the job stream via WebSocket:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)